playbooks/gpu_machineset_aws_template.json

```diff
@@ -0,0 +1 @@
ter-api-machineset": {}}}, "f:spec": {".": {}, "f:metadata": {}, "f:providerSpec": {".": {}, "f:value": {".": {}, "f:ami": {}, "f:apiVersion": {}, "f:blockDevices": {}, "f:credentialsSecret": {}, "f:deviceIndex": {}, "f:iamInstanceProfile": {}, "f:instanceType": {}, "f:kind": {}, "f:metadata": {}, "f:placement": {}, "f:securityGroups": {}, "f:subnet": {}, "f:tags": {}, "f:userDataSecret": {}}}}}}, "f:status": {}}, "manager": "cluster-bootstrap", "operation": "Update", "time": "2020-10-19T20:24:30Z"}, {"apiVersion": "machine.openshift.io/v1beta1", "fieldsType": "FieldsV1", "fieldsV1": {"f:metadata": {"f:annotations": {".": {}, "f:machine.openshift.io/GPU": {}, "f:machine.openshift.io/memoryMb": {}, "f:machine.openshift.io/vCPU": {}}}}, "manager": "machine-controller-manager", "operation": "Update", "time": "2020-10-19T20:32:51Z"}, {"apiVersion": "machine.openshift.io/v1beta1", "fieldsType": "FieldsV1", "fieldsV1": {"f:status": {"f:availableReplicas": {}, "f:fullyLabeledReplicas": {}, "f:observedGeneration": {}, "f:readyReplicas": {}, "f:replicas": {}}}, "manager": "machineset-controller", "operation": "Update", "time": "2020-10-19T20:42:42Z"}], "name": "ed-4-6-dev-gtpbh-worker-us-east-1a", "namespace": "openshift-machine-api", "resourceVersion": "21723", "selfLink": "/apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machinesets/ed-4-6-dev-gtpbh-worker-us-east-1a", "uid": "cf7074a6-dc42-4219-9592-e1c30d55485a"}, "spec": {"replicas": 1, "selector": {"matchLabels": {"machine.openshift.io/cluster-api-cluster": "ed-4-6-dev-gtpbh", "machine.openshift.io/cluster-api-machineset": "ed-4-6-dev-gtpbh-worker-us-east-1a"}}, "template": {"metadata": {"labels": {"machine.openshift.io/cluster-api-cluster": "ed-4-6-dev-gtpbh", "machine.openshift.io/cluster-api-machine-role": "worker", "machine.openshift.io/cluster-api-machine-type": "worker", "machine.openshift.io/cluster-api-machineset": "ed-4-6-dev-gtpbh-worker-us-east-1a"}}, "spec": {"metadata": {}, "providerSpec": {"value": {"ami": {"id": "ami-009cbe327d180d666"}, "apiVersion": "awsproviderconfig.openshift.io/v1beta1", "blockDevices": [{"ebs": {"encrypted": true, "iops": 0, "kmsKey": {"arn": ""}, "volumeSize": 120, "volumeType": "gp2"}}], "credentialsSecret": {"name": "aws-cloud-credentials"}, "deviceIndex": 0, "iamInstanceProfile": {"id": "ed-4-6-dev-gtpbh-worker-profile"}, "instanceType": "m4.large", "kind": "AWSMachineProviderConfig", "metadata": {"creationTimestamp": null}, "placement": {"availabilityZone": "us-east-1a", "region": "us-east-1"}, "securityGroups": [{"filters": [{"name": "tag:Name", "values": ["ed-4-6-dev-gtpbh-worker-sg"]}]}], "subnet": {"filters": [{"name": "tag:Name", "values": ["ed-4-6-dev-gtpbh-private-us-east-1a"]}]}, "tags": [{"name": "kubernetes.io/cluster/ed-4-6-dev-gtpbh", "value": "owned"}], "userDataSecret": {"name": "worker-user-data"}}}}}}, "status": {"availableReplicas": 1, "fullyLabeledReplicas": 1, "observedGeneration": 1, "readyReplicas": 1, "replicas": 1}}
```

Comment:
Agree with the bot, no EOL is a *bad* thing, same below.

Reply:
Ups those files are autogenerated should be there, removed